index.js

```diff
@@ -185,7 +196,7 @@ async function analyzeFile(cwd, file, options) {
   let content = fs.readFileSync(`${cwd}/${file}`, 'utf8');
   let extension = path.extname(file).toLowerCase();
 
-  if (extension === '.js') {
+  if (['.js', ...options.userExtensions].includes(extension)) {
```

Comment:
Just to make sure since I'm lacking a bit of context here. There are no extensions the user can provide which should be analyzed/compiled as `hbs` or `emblem`, right?

Reply:
Very late with this, but yeah we can only provide plugins for babel which will only affect the script files and not the template files such as hbs and emblem.